src/jquery.mockjax.js

```diff
@@ -115,6 +115,16 @@
 				return null;
 			}
 		} else {
+
+			// Apply namespace prefix to the mock handler's url.
+			var namespace = handler.namespace || $.mockjaxSettings.namespace;
+			if (!!namespace) {
+				handler.url = [
+						namespace,
+						handler.url
```

Comment:
Hmm... what if they prefix their handler url with "/"? I think we need to do a tiny bit more on this check to be smart about it.